tests/test_cli.py

```diff
@@ -85,3 +89,109 @@ def test_api_method_not_found(nyse_tickers, capfd):
     out, err = capfd.readouterr()
     assert 'null' in out
     assert f'No api method `{bad_meth}` could be found?' in err
+
+
+@pytest.fixture
+def temp_dir():
+    #Create temporary directory
+    with tempfile.TemporaryDirectory() as tempdir:
+        config_dir = os.path.join(tempdir, 'piker')
+        yield config_dir
+
+
+@pytest.fixture
+def piker_dir(temp_dir):
+    wl.make_config_dir(temp_dir)
+    path = os.path.join(temp_dir, 'watchlists.json')
+    yield path
+
+
+def test_watchlist_is_sorted_and_saved_to_file(piker_dir):
+    """Ensure that watchlist is sorted and saved to file
+    """
+    wl_temp = {'test': ['TEST.CN', 'AAA'], 'AA': ['TEST.CN']}
+    wl_sort = {'AA': ['TEST.CN'], 'test': ['AAA', 'TEST.CN']}
+    wl.write_sorted_json(wl_temp, piker_dir)
+    temp_sorted = wl.ensure_watchlists(piker_dir)
+    assert temp_sorted == wl_sort
+
+
+def test_watchlists_config_dir_created(caplog, temp_dir):
+    """Ensure that a config directory is created
+    """
+    with caplog.at_level(logging.DEBUG):
+        wl.make_config_dir(temp_dir)
+    assert len(caplog.records) == 1
+    record = caplog.records[0]
+    assert record.levelname == 'DEBUG'
+    assert record.message == f"Creating config dir {temp_dir}"
+    assert os.path.isdir(temp_dir)
+    #Test that there is no error and that a log message is not generatd
+    #when trying to create a directory that already exists
+    with caplog.at_level(logging.DEBUG):
+        wl.make_config_dir(temp_dir)
+    assert len(caplog.records) == 1
```

Comment:
Maybe add a small comment indicating that there should be no *additional* log messages emitted.